.github/workflows/qat.yml

```diff
@@ -52,19 +52,21 @@ jobs:
           fi
 
       - name: Check PMCQA
-        if: ${{ env.RUN_QAT == 'true' }} && ${{ inputs.THEME_NAME  != '' }} && ${{ inputs.SITE_URL != '' }}
```

Comment:
🤦 I missed this during review. This will work as long as all are in the same `${{ }}` block: https://github.com/penske-media-corp/github-action-wordpress-test-setup/blob/dccc527950ea012b22aeac07f987a35a3d3db0b5/action.yml#L147